pkg/result.go

```diff
@@ -46,3 +47,33 @@ func ChaosEngineVerdict(testsDetails *types.TestDetails, clients environment.Cli
 
 	return nil
 }
+
+func ChaosRunHistoryVerdict(testsDetails *types.TestDetails, clients environment.ClientSets, previousRunHistory v1alpha1.HistoryDetails) error {
+
+	if err = WaitForChaosResultCompletion(testsDetails, clients); err != nil {
+		return errors.Errorf("engine state check failed, err %v", err)
+	}
+
+	chaosResult, err := clients.LitmusClient.ChaosResults(testsDetails.ChaosNamespace).Get(testsDetails.EngineName+"-"+testsDetails.ExperimentName, metav1.GetOptions{})
+	if err != nil {
+		return errors.Errorf("Fail to get the chaosresult, due to %v", err)
+	}
+	// log.Infof("Chaos Result Verdict is: %v", chaosResult.Status.ExperimentStatus.Verdict)
+	// log.Infof("Chaos Run history is: %v", chaosResult.Status.History)
+	// log.Infof("Previous Chaos Run history is: %v", previousRunHistory)
```

Comment:
Can we uncomment them and let's use this in place of multiple prints.

```
	log.InfoWithValues("The chaos result run history is as follows", logrus.Fields{
		"Passed Runs": chaosResult.Status.History.PassedRuns,
		"Failed Runs":     chaosResult.Status.History.FailedRuns,
		"Stopped Runs": chaosResult.Status.History.StoppedRuns,
	})
```

The above one it just an example. You can use the values which you want to print.